projects/1.16/assets/fluid-system/fluidsystem/lang/zh_cn.json

```
@@ -0,0 +1,30 @@
+{
+    "block.fluidsystem.controller_tank_block": "储罐方块控制器",
+    "block.fluidsystem.infinite_water_source": "无限水源",
+    "block.fluidsystem.input_pipe": "输入管道",
+    "block.fluidsystem.input_tank_block": "储罐方块输入",
+    "block.fluidsystem.interface_tank_block": "储罐方块接口",
+    "block.fluidsystem.output_pipe": "输出管道",
+    "block.fluidsystem.pipe": "管道",
+    "block.fluidsystem.pipe_controller": "管道控制器",
+    "block.fluidsystem.structural_tank_block": "储罐方块",
+    "gui.fluidsystem.liquidAmount": "%smB/%smB",
+    "gui.fluidsystem.liquidPercentage": "流体： %s",
+    "gui.fluidsystem.tanksBlock": "储罐的方块： ",
+    "itemGroup.fluidsystem_tab": "流体系统",
+    "message.fluidsystem.fluidFilterRemoved": "移除流体过滤器",
+    "message.fluidsystem.fluidFiltered": "流体过滤： %s",
+    "message.fluidsystem.fluidFilteredSet": "流体过滤器设置： %s",
+    "message.fluidsystem.tankControllerMissing": "结构内缺少控制器",
+    "message.fluidsystem.tankExtraController": "结构内含有多个控制器",
+    "message.fluidsystem.tankFormed": "结构建造正确",
+    "message.fluidsystem.tankTooBig": "结构大小超过上限",
+    "messagefluidsystem.tankEmpty": "空",
+    "tooltip.fluidsystem.inputPipe": "你可以从管道中输入流体，同时发出红石信号",
+    "tooltip.fluidsystem.outputPipe": "允许从管道中输出流体",
+    "tooltip.fluidsystem.pipeController": "受到红石信号时会断开管道",
+    "tooltip.fluidsystem.tankBlock": "增加了储罐的容量",
+    "tooltip.fluidsystem.tankBlockController": "主要的方块，让你建造储罐结构",
+    "tooltip.fluidsystem.tankBlockInput": "允许流体通过管道的输入储罐",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
    "tooltip.fluidsystem.tankBlockInput": "该方块可以接收使用本模组管道运输而来的流体",
```